app/globals.css

```
@@ -65,10 +63,71 @@
   --tracking-body2: -0.105px;
   --tracking-caption: -0.09px;
 
+  /* Typography - Font Weight */
+  --font-weight-regular: 400;
+  --font-weight-semibold: 600;
+  --font-weight-bold: 700;
+
   /*layout */
```

Stylelint (17.2.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)